### PR TITLE
Tux: Fix default orbit style shown in status bar

### LINKED ISSUE #16360: [UI/UX] Orbit selector on Status bar not set correctly after fresh install
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

On a fresh installation the Orbit selector on the Status Bar is set to **Turntable**,
while in the Preferences Editor it's set to **Trackball**. The later corresponds to the the actual behavior.

Status bar: Turntable
Preferences: Trackball

### Full version info

```shell
OS: Debian GNU/Linux 12 (bookworm) (KDE/plasmawayland)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38622 (Git) AppImage
Build type: Release
Branch: main
Hash: ed97f3c4591303ee3b54906768f8f4eba5211265
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: English/United States (en_US)

&

[code]
OS: KDE Flatpak runtime (KDE/plasmawayland)
Word size of FreeCAD: 64-bit
Version: 0.21.2.33771 (Git)
Build type: Release
Branch: (HEAD detached at b9bfa5c)
Hash: b9bfa5c5507506e4515816414cd27f4851d00489
Python 3.11.7, Qt 5.15.10, Coin 4.0.0, Vtk 8.2.0, OCC 7.6.3
Locale: English/United States (en_US)
[/code]

&

Weekly Build on Windows 11
```

### Subproject(s) affected?

None

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Rexbas**
Fixes #16360.

### CODE REVIEW

**Review by hyarion — APPROVED** [review]
This change sets the [correct default value](https://github.com/FreeCAD/FreeCAD/blob/main/src/Gui/NavigationStyle.cpp#L162), in the same [hard coded way as the Preference dialog](https://github.com/FreeCAD/FreeCAD/blob/main/src/Gui/PreferencePages/DlgSettingsNavigation.ui#L478).

The underlying issue of these variables being hard coded to the same value is still there.
A better fix might be that we use the actual value from [the navigation style](https://github.com/FreeCAD/FreeCAD/blob/main/src/Gui/NavigationStyle.cpp#L162) as the default value, but that could cause worse issues if the order of initiation changes.

The only not-bad way I can come up with to always keep these in sync is if we create a constant in `Gui/NavigationStyle.h` with the default value and use that constant in these three places, but that would require some refactoring as the actual enum handling the orbital style is located [inside `Gui/NavigationStyle.cpp`](https://github.com/FreeCAD/FreeCAD/blob/main/src/Gui/NavigationStyle.cpp#L59) and needs to be exposed.

I think this is an acceptable fix for a current issue and it is also a fix that could be included in 1.0. Any fix to the underlying issue (if that is even needed) could be done after this has been merged.

I think we should approve and merge this pull request.

**Comment by Rexbas** [comment]
I agree that the hard coded default values are not perfect. It is not just navigation parameters but probably all configurable parameters in Tools -> Edit parameters... . I think the parameter system should be refactored to better support default values defined in a central place, in a way that both cpp and python can easily access it.

**Comment by chennes** [comment]
Thanks for the review, @hyarion, that's very helpful. Merged and cherry-picked to 1.0.

**Pull request merged**
